scripts/build.ps1

```diff
@@ -259,6 +265,20 @@ if ($Platform -eq "ios" -and !$Static) {
     Write-Host "iOS can only be built as static"
 }
 
+if (!$OfficialRelease) {
+    try {
+        $env:GIT_REDIRECT_STDERR = '2>&1'
+        # Thanks to https://stackoverflow.com/questions/3404936/show-which-git-tag-you-are-on
+        # for this magic git command!
+        $Output = git describe --exact-match --tags $(git log -n1 --pretty='%h')
```

Comment:
I think this requires the release tag to exist before the release build has even started. Also, the tag might not exist in the build machine's git repository when this is run.

Reply:
That's ok. The release tag **triggers** the official build for it. So the tag always is created first.